evap/staff/forms.py

```diff
@@ -917,20 +917,25 @@ class QuestionnairesAssignForm(forms.Form):
     def __init__(self, *args, course_types, **kwargs):
         super().__init__(*args, **kwargs)
 
-        for course_type in course_types:
-            self.fields[course_type.name] = forms.ModelMultipleChoiceField(
-                required=False,
-                queryset=Questionnaire.objects.general_questionnaires().exclude(
-                    visibility=Questionnaire.Visibility.HIDDEN
-                ),
-            )
         contributor_questionnaires = Questionnaire.objects.contributor_questionnaires().exclude(
             visibility=Questionnaire.Visibility.HIDDEN
         )
-        self.fields["all-contributors"] = forms.ModelMultipleChoiceField(
-            label=_("All contributors"), required=False, queryset=contributor_questionnaires
+        general_questionnaires = Questionnaire.objects.general_questionnaires().exclude(
+            visibility=Questionnaire.Visibility.HIDDEN
         )
 
+        for course_type in course_types:
+            self.fields["general-" + str(course_type.id)] = forms.ModelMultipleChoiceField(
```

Comment:
I think f-strings are more concise here, below, in the test and the other view
```suggestion
            self.fields[f"general-{course_type.id}"] = forms.ModelMultipleChoiceField(
```